NEWRELIC-5414 Updated the iOS agent upgrade docs (#12054)
newrelic/docs-website: src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/upgrade-new-relic-mobiles-ios-sdk.mdx

  - /docs/mobile-monitoring/new-relic-mobile-ios/install-configure/upgrade-new-relic-mobiles-ios-sdk-v4
  - /docs/mobile-monitoring/new-relic-mobile-ios/installation/upgrade-new-relic-mobiles-ios-sdk-v4/
---
 For information about the latest version, refer to the [release notes](/docs/releases/ios).

## Update with Swift Package Manager [#SPM]

To update the New Relic iOS agent using Swift Package Manager:

> 1. Navigate to the **NewRelic** agent in the Package Dependencies section of your xCode project navigator.

Reviewer: ```suggestion
1. Navigate to the **NewRelic** agent in the Package Dependencies section of your Xcode project navigator.
```

2. Right-click on the **NewRelic** agent.
3. Select **Update Package**.

## Update with Cocoa Pods [#Cocoa-Pods]

To update the New Relic iOS agent using Cocoa Pods:

1. Open the **Terminal** app and navigate to the directory with your PodFile.
2. Run the following command:
```
pod update NewRelicAgent
```

## Manually replace your iOS framework [#Manual]

To download and manually install the New Relic iOS agent:

1. Go to https://download.newrelic.com/ios_agent/ and download the iOS agent you want to update to.
2. Navigate to where the iOS agent was downloaded and unzip it.

> 3. From the Project Navigator (CMD 1) in Xcode, search for NewRelic.xcframework.

Reviewer 2: ```suggestion
3. From the Project Navigator (CMD 1) in Xcode, search for NewRelic.xcframework.
```
@mbruin-NR What is CMD 1? Are we suggesting a keyboard shortcut to get to the project navigator, or searching for NewRelic.xcframework?

Author: Yes the key command (CMD 1) will open the project navigator in xCode.

Reviewer 2: Okay sure, check out my suggestion ^

  - /docs/mobile-monitoring/new-relic-mobile-ios/install-configure/upgrade-new-relic-mobiles-ios-sdk-v4
  - /docs/mobile-monitoring/new-relic-mobile-ios/installation/upgrade-new-relic-mobiles-ios-sdk-v4/
---
 For information about the latest version, refer to the [release notes](/docs/releases/ios).

## Update with Swift Package Manager [#SPM]

To update the New Relic iOS agent using Swift Package Manager:

1. Navigate to the **NewRelic** agent in the Package Dependencies section of your xCode project navigator.
2. Right-click on the **NewRelic** agent.
3. Select **Update Package**.

## Update with Cocoa Pods [#Cocoa-Pods]

To update the New Relic iOS agent using Cocoa Pods:

1. Open the **Terminal** app and navigate to the directory with your PodFile.
2. Run the following command:
```
pod update NewRelicAgent
```

> 

Reviewer 2: ```suggestion
```

  - /docs/mobile-monitoring/new-relic-mobile-ios/install-configure/upgrade-new-relic-mobiles-ios-sdk-v4
  - /docs/mobile-monitoring/new-relic-mobile-ios/installation/upgrade-new-relic-mobiles-ios-sdk-v4/
---
 For information about the latest version, refer to the [release notes](/docs/releases/ios).

## Update with Swift Package Manager [#SPM]

Updating the New Relic iOS agent using Swift Package Manager:

> 1. Navigate to the NewRelic agent in the Package Dependencies section of the xCode project navigator.

Reviewer 2: ```suggestion
1. Navigate to the `NewRelic agent` in the Package Dependencies section of your xCode project navigator.
```

2. Right-click on the **NewRelic** agent.
3. Select **Update Package**.

## Update with Cocoa Pods [#Cocoa-Pods]

> Updating the New Relic iOS agent using Cocoa Pods:

Reviewer 2: ```suggestion
To update the New Relic iOS agent using Cocoa Pods:
```

1. Open the **Terminal** app and navigate to the directory with your PodFile.
2. Run the following command:
```
pod update NewRelicAgent
```

## Manually replace your iOS framework [#Manual]

> Downloading and manually installing the New Relic iOS agent.

Reviewer 2: ```suggestion
To download and manually install the New Relic iOS agent:
```

> ```suggestion

Reviewer 2: ```suggestion
```

> 1. Go to https://download.newrelic.com/ios_agent/ and select the iOS agent you want to update to. A download will be initiated. 

Reviewer 2: ```suggestion
1. Go to https://download.newrelic.com/ios_agent/ and download the iOS agent you want to update to.
```

> 1. First click this [link](https://download.newrelic.com/ios_agent/) to be taken to our iOS agent hosting site.
> 2. Click the version of the iOS agent you want to update to. This will begin the download.

Reviewer 2: ```suggestion
```suggestion
1. Go to https://download.newrelic.com/ios_agent/ and select the iOS agent you want to update to. A download will be initiated. 
```
```

> 3. Navigate to where the iOS agent was downloaded and unzip it.
> 4. From the Project Navigator (CMD 1) in Xcode, search for NewRelic.xcframework.
> 5. Right-click or control-click NewRelic.xcframework, and select Show in Finder.
> 6. Drag the newly downloaded iOS agent into the same folder as the old, and select to replace the old.

Reviewer 2: ```suggestion
2. Navigate to where the iOS agent was downloaded and unzip it.
3. From the Project Navigator (CMD 1) in Xcode, search for NewRelic.xcframework.
4. Right-click or control-click NewRelic.xcframework, and select **Show in Finder**.
5. Drag the newly downloaded iOS agent into the same folder as the old agent, and select to replace the old.
```

  - /docs/mobile-monitoring/new-relic-mobile-ios/install-configure/upgrade-new-relic-mobiles-ios-sdk-v4
  - /docs/mobile-monitoring/new-relic-mobile-ios/installation/upgrade-new-relic-mobiles-ios-sdk-v4/
---
 For information about the latest version, refer to the [release notes](/docs/releases/ios).

## Update with Swift Package Manager [#SPM]

To update the New Relic iOS agent using Swift Package Manager:

1. Navigate to the **NewRelic** agent in the Package Dependencies section of your Xcode project navigator.
2. Right-click on the **NewRelic** agent.
3. Select **Update Package**.

<Callout variant="important">

> Always check the [release notes](/docs/releases/ios) to make sure you are getting the most up to date agent versions. Depending on how your Swift Package Manager rules are set up you may need to change the `Up to Next Major` or Branch settings to include the new version.

Reviewer 2: ```suggestion
Always check the [release notes](/docs/releases/ios) to make sure you're getting the most up-to-date agent versions. Depending on how your Swift Package Manager rules are set up, you may need to change the `Up to Next Major` or Branch settings to include the new version.
```
@mbruin-NR "Branch" needs to either be lowercased or code formatted.
